ci-operator/step-registry/upi/vsphere/pre/upi-vsphere-pre-chain.yaml

```diff
@@ -2,7 +2,7 @@ chain:
   as: upi-vsphere-pre
   steps:
   - ref: ipi-install-rbac
-  - chain: upi-conf-vsphere
+  - chain: upi-conf-vsphere-platform-none-ovn
```

Comment:
I have updated the PR to reflect what we discussed, the only confusing bit to me atm is since I am updating the current [ upi-conf-vsphere-commands.sh](https://github.com/openshift/release/blob/master/ci-operator/step-registry/upi/conf/vsphere/upi-conf-vsphere-commands.sh) to remove platform specific information,  doesn't it technically also make the existing [upi-conf-vsphere](https://github.com/openshift/release/blob/master/ci-operator/step-registry/upi/conf/vsphere/upi-conf-vsphere-chain.yaml) chain which is using it to be included under the platform-none umbrella/ directory?